### PR TITLE
Add a ReportMsg interface to the SandboxOutput

### PR DESCRIPTION
**Pull request by trink**

### CODE REVIEW

File: sandbox/plugins/sandbox_output.go [hunk 1]
```diff
@@ -136,7 +137,6 @@ func (s *SandboxOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (
 				}
 			} else {
 				err = fmt.Errorf("FATAL: %s", s.sb.LastError())
-				or.LogError(err)
```
Comment (mreid-moz) [inline sandbox/plugins/sandbox_output.go]: Why not log this fatal error anymore?

Comment (trink) [inline sandbox/plugins/sandbox_output.go]: The error message will be output twice, which is just annoying.

File: sandbox/plugins/sandbox_output.go [hunk 2]
```diff
@@ -171,6 +170,42 @@ func (s *SandboxOutput) Run(or pipeline.OutputRunner, h pipeline.PluginHelper) (
...
+	if s.processMessageSamples > 0 {
+		tmp = s.processMessageDuration / s.processMessageSamples
+	}
+	message.NewInt64Field(msg, "ProcessMessageAvgDuration", tmp, "ns")
+
+	tmp = 0
+	if s.timerEventSamples > 0 {
+		tmp = s.timerEventDuration / s.timerEventSamples
+	}
+	message.NewInt64Field(msg, "TimerEventAvgDuration", tmp, "ns")
```
Comment (mreid-moz) [inline sandbox/plugins/sandbox_output.go]: Consider introducing an "avg" utility function to save duplicate code.

**Comment by mreid-moz** [comment]
Looks good to me. Up to you whether it's worthwhile adding a function to compute average (and avoid div by 0)

File: sandbox/plugins/sandbox_output.go (hunk 2 above)
Comment (trink) [inline sandbox/plugins/sandbox_output.go]: I will leave it like the others for now... but noting it for https://github.com/mozilla-services/heka/issues/918

**Pull request merged**